message.go

```diff
@@ -0,0 +1,20 @@
+package sequence
+
+import (
+	"bytes"
+	"fmt"
+)
+
+func IsEIP191Message(msg []byte) bool {
+	return len(msg) > 0 && msg[0] == 0x19
```

Comment:
I think you should check for the full prefix of `"\x19Ethereum Signed Message:\n"` via bytes.HasPrefix